sessionctx/variable/sysvar.go

```diff
@@ -457,6 +458,44 @@ var defaultSysVars = []*SysVar{
 			return BoolToOnOff(config.GetGlobalConfig().Instance.TiDBEnableDDL.Load()), nil
 		},
 	},
+	{Scope: ScopeInstance, Name: TiDBTmpStoragePath, Value: config.GetGlobalConfig().TempStoragePath, Type: TypeStr, SetGlobal: func(s *SessionVars, val string) error {
+		// FIXME: check if temp dir is being used; if used, reject
```

Comment:
In MySQL this feature would be called `tmpdir`. It's currently a noop, but can be changed to an actual variable -- there is no reason we can't use the MySQL name here because the purpose is the same.